src/sentry/static/sentry/app/actionCreators/projects.tsx

```diff
@@ -48,13 +58,13 @@ export function loadStats(api, params) {
 
 // This is going to queue up a list of project ids we need to fetch stats for
 // Will be cleared when debounced function fires
-const _projectStatsToFetch = new Set();
+const _projectStatsToFetch: Set<string> = new Set();
```

Comment:
Could also just do `const _projectStatsToFetch = new Set<string>();

Reply:
How would it know the contents of the set?